src/commands/package.command.ts

```diff
@@ -6,18 +6,27 @@ export class PackageCommand {
     private packageManagerFactory = new PackageManagerFactory();
 
     public async pullPackage(profile: string, key: string, store: boolean, newKey: string): Promise<void> {
-        await this.contentService.pullFile(profile, this.packageManagerFactory.createManager(key, null, store, newKey));
+        await this.contentService.pullFile(
+            profile,
+            this.packageManagerFactory.createManager(key, null, null, store, newKey)
+        );
     }
 
-    public async pushPackage(profile: string, fileName: string, newKey: string, overwrite: boolean): Promise<void> {
+    public async pushPackage(
+        profile: string,
+        spaceId: string,
```

Comment:
Why di we use here a `spaceId` while in the push commands we're use `spaceKey`?

Reply:
Ah, I forgot it there, It's just the parameter name I'll update it 👍